scripts/fix-engine-names.py

```diff
@@ -22,6 +22,21 @@
     "Stockfish_13_CCRL 64-bit_4CPU": ("Stockfish", "13", "CCRL 64-bit 4CPU"),
     "Stockfish_15_CCRL 64-bit_4CPU": ("Stockfish", "15", "CCRL 64-bit 4CPU"),
     "Chat": ("TCEC Chat", "", ""),
+    "EtherealClassical 12.89": ("Ethereal", "12.89_Classical", ""),
+    "SFNNUE 20200704-StockFiNN-0.2": ("Stockfish", "20200704-StockFiNN-0.2", "SFNNUE"),
+    "StockfishNNUE 20200704-StockFiNN-0.2": (
+        "Stockfish",
+        "20200704-StockFiNN-0.2",
+        "StockfishNNUE",
+    ),
+    "SimpleEval2 20200731r14": ("SimpleEval", "20200731r14", "2"),
+    "Bluefish Dev": ("Stockfish", "Dev", "Bluefish"),
+    "Booot2 6.5": ("Booot", "6.5", "2"),
+    "RubiChessClassical 2.0.1": ("RubiChess", "2.0.1_Classical", ""),
+    "StockfishClassical": ("Stockfish", "Classical", ""),
```

Comment:
Just a note: It is unfortunate that we have engines in the PGNs without version info. The actual engine version should be traced if it can be done. (But that can be done later. Besides, it requires special treatment since it's event-specific.)